web/cobrands/bathnes/staff.js

```diff
@@ -6,8 +6,7 @@ if (!fixmystreet.maps || !fixmystreet.maps.banes_defaults) {
 
 var llpg_stylemap = new OpenLayers.StyleMap({
     'default': new OpenLayers.Style({
-        fillColor: "#FFFF00",
-        fillOpacity: 0.6,
+        fillOpacity: 0,
```

Comment:
This could be `fill: false`, but meh, makes no difference to the visible end result 😃